<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchControlPage</class>
 <widget class="QWidget" name="SwitchControlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#titleLabel {
	padding-width-left:5px;
	padding-width-right:5px;
	border: 1px solid white;
	border-radius: 10px;
	min-width: 200px;
	min-height: 40px;
}

QFrame#frame_1,#frame_2,#frame_3  {
	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 3px;
	background-color:rgb(100, 200, 200, 50%);
}

QPushButton {
	min-width: 100px;
	min-height: 100px;
	max-width: 100px;
	max-height: 100px;
	border-image: url(:/Hac/round-button.png);
}


 QPushButton#scheduleButton {
	image: url(&quot;:/Hac/clock.png&quot;);
 }

QPushButton#numUpButton, #numDownButton {
	min-width: 55px;
	min-height:55px;
	max-width: 55px;
	max-height: 55px;
	border-image: url(:/Hac/round-button-small.png);
}

QPushButton#numUpButton {
	image: url(:/Hac/up-arrow-black.png);
}
QPushButton#numUpButton:pressed {
	image: url(:/Hac/up-arrow-red.png);
}
QPushButton#numDownButton {
	image: url(:/Hac/down-arrow-black.png);
}
QPushButton#numDownButton:pressed {
	image: url(:/Hac/down-arrow-red.png);
}


QLabel#lightPixmapLabel {
	min-width: 80px;
	min-height: 80px;
	max-width: 80px;
	max-height: 80px;

	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 10px;
	background-color:rgb(100, 200, 200, 50%);

}

QSpinBox {
	margin: 1px;
	selection-background-color: transparent;
	text: black;
	selection-color: black;
	font: normal 15px;
	border: 1px solid black;
	padding-width-right: 40px;
}

QSpinBox::up-button {
	subcontrol-origin: border;
    subcontrol-position: right center;
	width: 20px;
	height: 20px;
}

QSpinBox:up-arrow {
     image: url(:/Hac/up-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::up-arrow:pressed {
     image: url(:/Hac/up-arrow-red.png);
}


QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: right center;
	left: -20px;
	width: 20px;
	height: 20px;
}

QSpinBox::down-arrow {
     image: url(:/Hac/down-arrow-black.png);
     width: 20px;
     height: 20px;
}
QSpinBox::down-arrow:pressed {
     image: url(:/Hac/down-arrow-red.png);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Switch Control</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Control
All</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="allOnButton">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="allOffButton">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_1">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>No.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="numDigits">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="numDownButton">
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="numUpButton">
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>38</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lightPixmapLabel">
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="lightOnButton">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lightOffButton">
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/hac.qrc"/>
 </resources>
 <connections/>
</ui>
